src/pybind/mgr/requirements-mgr.in

```diff
@@ -0,0 +1,14 @@
+wheel==0.31.1
+cherrypy==18.8.0
+pyjwt==2.4.0
+routes==2.5.1
+werkzeug==2.0.3
+saml==0.7.0
+setuptools==59.6.0
+setuptools_scm>=1.15.7
+importlib-metadata==4.8.3
+importlib-resources==5.4.0
+pkgconfig>=1.5.1
+zipp==3.6.0
+typing-extensions==4.1.1
+dataclasses==0.6 # from werkzeug
```

Comment:
Interesting, so we had `dataclassses` available?

Reply:
`werkzeug` had dataclasses as a dependency so looks like it :)

Reply:
I've updated it too since I don't see why we shouldn't have pretty much all requirements like this since it enables bug fixes to be updated